platform/import-export/src/actions/ExportAsJSONAction.js

```diff
@@ -121,8 +121,8 @@ define(['lodash'], function (_) {
      */
     ExportAsJSONAction.prototype.rewriteLink = function (child, parent) {
         this.externalIdentifiers.push(this.getId(child));
-        var index = _.findIndex(parent.composition, function (id) {
-            return _.isEqual(child.identifier, id);
+        var index = parent.composition.findIndex(function (id) {
+            return JSON.stringify(child.identifier) === JSON.stringify(id);
```

Comment:
Let's not re-implement `_.isEqual` right now, it's actually pretty involved. This is a good case where using a library actually makes sense. The implementation here is not equivalent to `_.isEqual`

eg.
```JavaScript
const objA = {propA: 'valueA', propB: 'valueB'};
const objB = {propB: 'valueB', propA: 'valueA'};
_.isEqual(objA, objB);
// true
JSON.stringify(objA) === JSON.stringify(objB);
//false
```

Please restore `_.isEqual` wherever it was being used.

Reply:
This is the only place I replaced this one. Seemed like a tempting place to nuke the import. Restored the lodash method.